packages/cli/src/commands/domains/index.ts

```diff
@@ -45,6 +45,7 @@ const help = () => {
   )}        Login token
     -S, --scope                    Set a custom scope
     -N, --next                     Show next page of results
+    -y, --yes                      Skip questions when setting up new project using default scope and settings
```

Comment:
Is there a “confirm” option for domains?

I’m not sure why “yes” was added here. Where is it used?

Reply:
Good catch! The description is inaccurate in this case. It's used here: https://github.com/vercel/vercel/blob/main/packages/cli/src/commands/domains/rm.ts#L81.